js/utils.js

```diff
@@ -0,0 +1,35 @@
+export default class Utils {
+	static getStorageItem(key) {
+		const data = window.localStorage.getItem(key);
+		if (typeof data === 'string') {
+			try {
+				return JSON.parse(data);
+			} catch (e) {
+			}
+		}
+		return data;
```

Comment:
```suggestion
		return JSON.parse(window.localStorage.getItem(key));
```
localStorage에 저장되는 타입은 무조건 string입니다. 그래서 그냥 단순하게 이렇게 표현해도 된답니다!